x11/luit/Portfile

```diff
@@ -1,7 +1,7 @@
 PortSystem          1.0
 
 name                luit
-version             2.0.20240102
+version             2.0.20250912
 categories          x11
 license             X11
 platforms           darwin
```

Comment:
this is the default and the line should thus be removed (as `port lint` warns about).